web/app.php

```diff
@@ -1,10 +1,18 @@
 <?php
-use LibraryLoaderAutoloader,
-    LibraryControllerFrontController;
-    
-require_once __DIR__ . "/Library/Loader/Autoloader.php";
-$autoloader = new Autoloader;
-$autoloader->register();
-
-$frontController = new FrontController();
-$frontController->run();
\ No newline at end of file
+
+require_once __DIR__ . './../vendor' . '/autoload.php';
+require_once __DIR__ . './../app/config' . '/services.php';
```

Comment:
De autoload en de services dienen door de front controller te worden opgepakt. 

De vendor autoload is niet verantwoordelijk voor de services. Je kan ze echter wel in een aparte file samen voegen. Want je heb deze bestanden nodig bij PHPUnit, Command Line opties en mogelijk andere hulp tools.